01-JS-DOM-APIs/Ex03/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+   <head>
+		<meta charset="UTF-8">
+		<meta http-equiv="x-ua-compatible" content="ie=edge">
+   	<title> JS 03 </title>
+   	<meta name="description" content="Let's begin with JavaScript">
+   	<meta name="viewport" content="width=device-width, initial-scale=1">
+      <link rel="stylesheet" href="css/styles.css">
+      <link href="https://fonts.googleapis.com/css?family=Bubbler+One" rel="stylesheet">
+   </head>
+   <body>
+   	<header>
+   		<h1>This it my third JS excercise!</h1>
+   	</header>
+   	<section id="jokeTextbox"></section> <br>
+      <button onclick="getJoke('http://api.icndb.com/jokes/random')"> I have a joke for you </button>
+      <script>
+         function getJoke(url) {
+            var joke;
+            var jokeRequest = new XMLHttpRequest();
+            jokeRequest.onreadystatechange = function() {
+            if (this.readyState == 4 && this.status == 200) {
```

Comment:
At first look, I thought that `jokeRequest.open("Get", url, true)` and the next sentence were outside the function, which is no posible. Try to make the code clearer, for example:

```
let getJoke = function (url) {
    let joke; //We should use es6
    let jokeRequest = new XMLHttpRequest();
   
    jokeRequest.onreadystatechange = function () {
    
        if (this.readyState === 4 && this.status === 200) { //Take about the equal comparision, I recommend you to use the deep comparision instead
             joke = JSON.parse(this.responseText);
             document.getElementById('jokeTextbox').innerHTML = joke.value.joke; //Always use single quoutes for javascript code. The double quotes are for html
         }
    }
   
    jokeRequest.open('GET', url, true);
    jokeRequest.send();
}
```